test-suite/src/test/scala/org/scalajs/testsuite/javalib/math/BigIntegerNotTest.scala

```diff
@@ -1,3 +1,4 @@
+// scalastyle:off
```

Comment:
I recall a really old coding standard that said something like "Don't add comments such as `// TODO: fix style` but rather just fix it now." 

 I you wish, I can do this _now_ - may mean you postpone a few hours